shell.nix

```diff
@@ -5,15 +5,16 @@ let
   inherit (nixpkgs) pkgs;
 
   f = { mkDerivation, base, containers, directory, errors, filepath
-      , generic-deriving, lens, mtl, stdenv, text, transformers
+      , generic-deriving, lens, mtl, stdenv, text, transformers, wl-pprint-text
+      , hspec, hspec-expectations-pretty-diff
```

Comment:
Why those new dependencies? I don't see them used anywhere.

Reply:
The build fails without them, the tests require them apparently (hspec is a testing framework).

Reply:
I guess due to some library update. Ok, perfect :-)